.github/workflows/fake-bpy-ci.yml

```
@@ -0,0 +1,84 @@
+name: fake-bpy-module CI
+
+on:
+  push:
+    branches:
+      # Temporarily trigger only on "ci_testing" branch
+      - ci_testing
+  # # Trigger on all PRs. By default, only run if activity type is opened, synchronize, or reopened
+  # pull_request:
+
+jobs:
+  build_modules:
+    name: Build fake-bpy modules
```

The project is `fake-bpy-module`, so I'm appreciate to change this name to fake-bpy-module.

```suggestion
    name: Build fake-bpy-module
```

---

I called it "modules" because we are building multiple modules:
fake-bpy-module-2.79, fake-bpy-module-2.80, fake-bpy-module-2.81, ...

I tried `name: Build fake-bpy-module modules` but it sounded weird.

When I implement the  "Matrix" feature (currently WIP), then it will be renamed to `name: build fake-bpy ${blender_version}` (or similar).

---

Then, how about `Build modules` instead?
I don't want to name this modules because we can refer this module such as `bpy` in Python not `fake-bpy`.

---

changed